qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -52,11 +52,7 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
         if "random_state" not in kwargs:
             kwargs["random_state"] = algorithm_globals.random_seed
 
-        super().__init__(
-            kernel=self._quantum_kernel.evaluate,
-            *args,
-            **kwargs,
-        )
+        super().__init__(kernel=self._quantum_kernel.evaluate, *args, **kwargs)
```

Comment:
`make black` is automatically reformatting this for me. I suppose I should leave it and allow it to be merged into main?